packages/kit/src/core/build/index.js

```diff
@@ -179,42 +221,42 @@ async function build_client({
 
 	print_config_conflicts(conflicts, 'kit.vite.', 'build_client');
 
-	await vite.build(merged_config);
+	const output = /** @type {import('rollup').OutputChunk[]} */ (
+		/** @type {any} */ (await vite.build(merged_config)).output
+	);
 
-	const client_manifest_file = `${client_out_dir}/manifest.json`;
 	/** @type {import('vite').Manifest} */
-	const client_manifest = JSON.parse(fs.readFileSync(client_manifest_file, 'utf-8'));
-	fs.renameSync(client_manifest_file, `${output_dir}/manifest.json`); // inspectable but not shipped
+	const manifest = JSON.parse(fs.readFileSync(`${client_out_dir}/manifest.json`, 'utf-8'));
 
-	return client_manifest;
+	// TODO where does this go?
```

Comment:
I don't understand this comment. Perhaps it could be expanded upon?

Reply:
was just an out of date comment — removed